CalibTracker/SiStripDCS/test/ParseManualO2Olog.py

```diff
@@ -1308,7 +1308,7 @@ def CompareReducedDetIDs(FirstDict,SecondDict):
         #    print row
         #if row['change_date']==datetime.datetime(2010, 8, 28, 22, 51, 8, 994000):
         #    print row
-        if row['change_date']>datetime.datetime(2010, 8, 28, 22, 44) and row['change_date']<datetime.datetime(2010, 8, 28, 23, 05,37):
+        if row['change_date']>datetime.datetime(2010, 8, 28, 22, 44) and row['change_date']<datetime.datetime(2010, 8, 28, 23, 0o5,37):
```

Comment:
well, the representation does not matter. The actual value is the same...

````
>>> oct(5)
'05'
````

It was a misuse of the octal representation...